common/src/test/java/io/seata/common/loader/EnhancedServiceLoaderTest.java

```diff
@@ -98,4 +98,32 @@ public void getAllExtensionClass1() {
         assertThat(allExtensionClass).isNotEmpty();
     }
 
+    @Test
+    public void getSingletonExtensionInstance(){
+        Hello hello1 = EnhancedServiceLoader.load(Hello.class, "ChineseHello");
+        Hello hello2 = EnhancedServiceLoader.load(Hello.class, "ChineseHello");
+        assertThat(hello1 == hello2);
+    }
+
+    @Test
+    public void getMultipleExtensionInstance(){
+        Hello hello1 = EnhancedServiceLoader.load(Hello.class, "LatinHello");
+        Hello hello2 = EnhancedServiceLoader.load(Hello.class, "LatinHello");
+        assertThat(hello1 != hello2);
```

Comment:
LatinHello is SINGLETON not PROTOTYPE.

Reply:
My fault , add scope = prototype in LatinHello